### ISSUE TITLE
CIFAR100 accuracy

### ISSUE THREAD

**Issue by gergopool** [KyleZheng1997/ReSSL#12]
After downloading your tiny-ressl implementation and evaluating your pretrained cifar100 network, I get 66.7%, while you reported 63.8% in the paper. When you have the time, can you please double-check which percentage is right? Thanks.

**Comment by mingkai-zheng** [comment]
We only pre-train the model for 200 epochs in our paper. The code and pre-trained model that we provided in this repository is for 800 epochs. For more details about CIFAR-100, please check the discussion in [Issue 8](https://github.com/kyle-1997/ReSSL/issues/8#issue-1100077730)
